producer/src/main/java/com/internship/producer/controller/ProductController.java

```diff
@@ -0,0 +1,30 @@
+package com.internship.producer.controller;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.internship.producer.service.ProductService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.RequestBody;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+import store.bean.Product;
+
+@RestController
+@RequestMapping("/product")
```

Comment:
It is better to store constants in a separate class or interface. And then, when creating a controller, use static imports. This is necessary so that your tests and controllers use the same constants.